.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -245,7 +245,7 @@ def process_and_execute_notebook(
         result.logs_bucket = operation_metadata.build.logs_bucket
 
         # Block and wait for the result
-        operation_result = operation.result(timeout=86400)
+        operation_result = operation.result(timeout=timeout_in_seconds)
```

Comment:
Instead of hardcoding (which would be out of date if the desired timeout changes), it should reflect what the Cloud Build job's timeout.